urllib3/poolmanager.py

```diff
@@ -227,8 +227,8 @@ def __init__(self, proxy_url, num_pools=10, headers=None,
             port = port_by_scheme.get(proxy.scheme, 80)
             proxy = proxy._replace(port=port)
 
-        assert proxy.scheme in ("http", "https"), \
-            'Not supported proxy scheme %s' % proxy.scheme
+        if proxy.scheme not in ("http", "https"):
+            raise ValueError('Not supported proxy scheme %s' % proxy.scheme)
```

Comment:
90% certain we don't, but others might. It's backwards incompatible because if someone has

```py
try:
    proxied_http = ProxyManager(url)
except AssertionError:
    pass
```

To catch the case where this exception is raised, then when they upgrade, the next time they see a proxy url that would have triggered that, they'll have an uncaught exception.